dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -48,6 +48,13 @@ def create
     render json: section.summarize
   end
 
+  # DELETE /api/v1/sections/<id>
+  # Delete a section
+  def destroy
+    @section.destroy
+    head :no_content
```

Comment:
I think it's needed, otherwise the default render will look for a view and fail.